src/tests/ftest/rebuild/read_array.yaml

```diff
@@ -10,6 +10,8 @@ hosts:
 timeout: 240
 server_config:
   name: daos_server
+  servers:
+    targets: 1
```

Comment:
This appears to be masking a problem.  Does the test fail if we use 8 targets?  If so then this test should be at least skipped for a ticket.  Since this test class only has one test method, we could implement a `!mux` here and include a `CANCEL_FOR_TICKET = [["DAOS-xxxx", "targets", 8]]` if we want to continue to run the test with 1 target but skip running with 8.

Reply:
Please see the new commit.